From 4492562c502cdc01ba6293e8667cbc08fd5de1be Mon Sep 17 00:00:00 2001
From: DDoSolitary <DDoSolitary@gmail.com>
Date: Sun, 29 Nov 2020 18:16:01 +0800
Subject: [PATCH] Fix compatibility with GCC 10.

GCC 10 defaults to -fno-common, which requires "extern" when defining
global variables in headers.

Signed-off-by: DDoSolitary <DDoSolitary@gmail.com>
---
 src/datastructure.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/datastructure.h b/src/datastructure.h
index e92adbe8..8592a7d9 100644
--- a/src/datastructure.h
+++ b/src/datastructure.h
@@ -16,7 +16,7 @@
 // enum privacy_level
 #include "enums.h"
 
-const char *querytypes[TYPE_MAX];
+extern const char *querytypes[TYPE_MAX];
 
 typedef struct {
 	unsigned char magic;
